TestTools/StreamChatTestTools/Mocks/StreamChat/Workers/ChannelUpdater_Mock.swift

```diff
@@ -250,6 +252,12 @@ final class ChannelUpdater_Mock: ChannelUpdater {
     override func deleteChannel(cid: ChannelId, completion: ((Error?) -> Void)? = nil) {
         deleteChannel_cid = cid
         deleteChannel_completion = completion
+        if let result = deleteChannel_completion_next_result {
```

Comment:
For async functions I want to feed in the next result before calling the function. Our current tests with completion handler call the function first, and then access the deleteChannel_completion later. In summary, the order changes with async function testing.